copr-builder.py

```diff
@@ -18,6 +18,7 @@
 
 Version = namedtuple('Version', ['version', 'build', 'date', 'git_hash'])
 
+BUILD_URL_TEMPLATE = "https://copr.fedorainfracloud.org/coprs/%s/%s/build/%s"
```

Comment:
Unfortunately this url doesn't work for group projects. For groups it looks like this:

`https://copr.fedorainfracloud.org/coprs/g/storage/udisks-experimental/build/534732/`

Group projects should be easy to identify -- `copr_user` starts with `@`